lib/archethic/oracle_chain/services/hydrating_cache.ex

```diff
@@ -132,4 +134,12 @@ defmodule Archethic.OracleChain.Services.HydratingCache do
   def handle_info(:discard_value, state) do
     {:noreply, %State{state | value: nil, ttl_timer: nil}}
   end
+
+  defp next_tick_in_seconds(refresh_interval) do
+    if is_binary(refresh_interval) do
```

Comment:
You should leverage `CronParser.parse(extended_mode)` instead, it's more accurate.

Reply:
As there is not proper error handling, I think let it crash would be ok.

Reply:
as discussed, it is fine to let it crash.